test/conformance/kuttl/generate/policy/cornercases/pol-data-create-on-trigger-deletion/01-manifests.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: pol-create-on-trigger-deletion-ns
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-org
+  namespace: pol-create-on-trigger-deletion-ns
+---
+apiVersion: kyverno.io/v1
```

Comment:
This is v1 while the other was v2beta1. Let's make these all consistent through this PR.